TrafficCapture/trafficCaptureProxyServer/src/test/java/org/opensearch/migrations/trafficcapture/proxyserver/testcontainers/annotations/KafkaContainerTest.java

```diff
@@ -1,10 +1,16 @@
 package org.opensearch.migrations.trafficcapture.proxyserver.testcontainers.annotations;
 
+import java.lang.annotation.ElementType;
 import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 import org.junit.jupiter.api.parallel.ResourceLock;
 
 @Inherited
 @ResourceLock("KafkaContainer")
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE})
```

Comment:
What are these lines doing here?  Why do you need them?

Reply:
Java by default only adds annotations to the .class file and are not loaded at runtime, for Junit Tags to work, these annotations need to be included at runtime.